translations/en.json

```diff
@@ -1687,6 +1687,8 @@
     "database-reset-content": "Chats, contacts and settings have been deleted. You can use your account with your Keycard",
     "database-reset-warning": "Database will be reset. Chats, contacts and settings will be deleted",
     "empty-keycard-required": "Requires an empty Keycard",
+    "empty-keycard": "Empty Keycard",
+    "user-keycard": "{{name}} Card",
```

Comment:
Is this actually what the designers envisioned or should it just be the users name? cc @xAlisher